src/main/java/uk/gov/beis/els/util/FontUtils.java

```diff
@@ -0,0 +1,61 @@

```

Comment:
I thought this was just going to be `a-Z 0-9`. Given its so large it would be nice to move this map to a file (since its just reference data) and read it in at runtime. It could also couldn't be stored in an enum since its not a compile time constant. 

Given it works as-is though I don't feel to strongly, and not worth delaying the release over. 
If you agree its worth changing it I could have a quick look tomorrow morning and see if I can get it loading form a file in an hour or 2, and if not we can just go with it as is?

Reply:
Maybe a stupid question but just to check is `.put('�', 0f)` intended or should that have been an actual resolvable unicode char? I notice its not in the other 2 font tables